.travis.yml

```diff
@@ -7,3 +7,4 @@ script:
   - "DJANGO_SETTINGS_MODULE='tests.settings' coverage run ./manage.py django test"
   - "coverage report -m --fail-under=80"
   - "flake8 --max-line-length=150 --statistics openslides tests"
+  - "isort --line-width 120 -rc -m 4  -c ."
```

Comment:
I would prefer the following line:
`"isort --line-width 115 --multi_line 3 --check-only --recursive openslides tests"`

Alternative: `"isort --line-width 115 --multi_line 1 --check-only --recursive openslides tests"`

* The long option flags are a little bit more understandable so that we do not have to read the isort docs if we look later at this line.
* I propose to move the recursive flag to the end just before the file definitions because it belongs more to it than the other flags.
* Lines of 120 are a little bit too long for GitHub diff view.
* The file definition `.` is not possible because it also catches files in a virtualenv folder and such stuff.
* Last but not least I like a vertical style more than  the grid style because it is easier to see the changes in a diff later on.